lms/templates/navigation-edx.html

```diff
@@ -79,9 +79,7 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
         <a href="#" class="dropdown" aria-haspopup="true" aria-expanded="false"><span class="sr">${_("More options dropdown")}</span> &#9662;</a>
         <ul class="dropdown-menu" aria-label="More Options" role="menu">
           <%block name="navigation_dropdown_menu_links" >
-            % if settings.MKTG_URL_LINK_MAP.get('FAQ'):
```

Comment:
I agree that this shouldn't be removed in this PR. If it needs to be removed for another reason then it should be done separately.